src/modules/location/index.ts

```diff
@@ -1140,8 +1153,21 @@ export class LocationModule {
     options?:
       | [latitude: number, longitude: number]
       | {
+          /**
+           * The original coordinate to get a new coordinate close to.
+           */
           origin?: [latitude: number, longitude: number];
```

Comment:
Shouldn't this be `[faker.location.latitude(), faker.location.longitude()]` instead?

Reply:
This the type definition not the default definition.
The origin parameter doesnt have a default.
Or did I missunderstood you?

Reply:
The origin does indeed habe a default value:

https://github.com/faker-js/faker/blob/5410239245b4a6fe8c1976f8aa33c970923f9f40/src/modules/location/index.ts#L1179-L1182

Thats why I was asking why the default was no provided in the summary like this:

```suggestion
          /**
          * The original coordinate to get a new coordinate close to.
          *
          * @default `[faker.location.latitude(), faker.location.longitude()]`
          */
          origin?: [latitude: number, longitude: number];
```

Reply:
> The origin does indeed habe a default value:

Close, but no. The option doesn't have a default. 
If the option is missing a different code path is chosen e.g. returning a random location instead of a location close to that origin location.

Reply:
I'm stupid, I see. I thought the `return [...]` was assigning the value like `origin = [...]`. Sorry for the confusion.